src/containers/About/About.js

```diff
@@ -174,33 +98,58 @@ export /* istanbul ignore next */ class About extends Component {
           <Table
             title="Dashboard"
             headers={headers}
-            rows={filteredRows(dashboardPropertiesToCheck)}
-            loading={loading}
+            rows={[
+              getRow('Namespace', this.props.dashboardNamespace),
+              getRow('Version', this.props.dashboardVersion),
+              getRow('IsOpenShift', this.props.isOpenShift),
+              getRow('ReadOnly', this.props.isReadOnly),
+              getRow('LogoutURL', this.props.logoutURL)
+            ].filter(item => item)}
```

Comment:
`.filter(Boolean)` is a shorthand for this that we've used elsewhere including in this file

Reply:
fixed